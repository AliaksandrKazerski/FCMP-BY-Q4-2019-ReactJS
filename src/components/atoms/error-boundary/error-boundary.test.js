import React from 'react';
import { shallow } from 'enzyme';

import ErrorBoundary from './index';

describe('ErrorBoundary', () => {
  it("ErrorBoundary should render correctly", () => {
  const myComponent = shallow(<ErrorBoundary/>);
  
  expect(myComponent).toMatchSnapshot()
  });
});