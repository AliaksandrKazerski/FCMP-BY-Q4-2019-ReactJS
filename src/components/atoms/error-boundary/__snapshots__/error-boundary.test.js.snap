// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `ShallowWrapper {}`;

exports[`ErrorBoundary ErrorBoundary should render correctly 1`] = `ShallowWrapper {}`;

exports[`News container check DatePicker popup open 1`] = `ShallowWrapper {}`;
